### ISSUE TITLE
chimp tests all features, regardless of the @watch tag

### ISSUE THREAD

**Issue by preexo** [xolvio/chimp#373]
I am expecting just wrong usage from my side, but it's not working as expected, so I thought creating an issue might help.

#####  Expected behavior
Chimp only tests the features that have the @watch tag.

#####  Actual behavior
Chimp runs through all features in the features directory

##### Steps to reproduce the problem
We run chimp with this `sudo chimp --ddp=http://localhost:3000 --watch --path=tests --browser=phantomjs`

#####  Used version

* Chimp: 0.33.1
* Node.js: v5.11.0
* Operation system: OSX 

#####  Console / Log Output

Chimp basically runs through all features in the feature folder, even though only one feature file has the @watch tag.

**Comment by samhatoum** [comment]
hey. where are you putting the watch tag?

**Comment by preexo** [comment]
In the *.feature file, on top of everything:
```
@watch
Feature: Create a Widget

 As a visitor to the site,
 so that I can manage widgets,
 I want to create and delete widgets on my home page.

 Background:
    Given I am on the site

 Scenario: Visitor creates a widget
    When I name a widget "alpha"
    And submit the form
    Then I should see a list of widget names containing "alpha"
```
In the same folder I have many more feature files, most of them are missing the steps actually and chimp runs through all of them. I would have expected that it only tests the feature file which has the @watch tag.

**Comment by preexo** [comment]
Ok so I found some time to investigate and it was actually something really simple. We had one feature file which actually contained two features, like this `multiple.feature`
```
Feature: Feature 1
Scenario: Scenario 1
Given Something is given
When When something is being done
Then Then something should happen

Feature: Feature 2
Scenario: Scenario 2
Given Something is given
When When something is being done
Then Then something should happen
```

When I removed the second feature, chimp wouldn't run any tests because there was no watch tag on any feature:
```
[chimp] Running...
0 scenarios
0 steps
0m00.000s
```

But when I added the second feature to the `multiple.feature` file it started running through all tests that existed, regardless of watch tags or not.

Is this totally wrong usage alone or also some kind of bug in chimp?

We will just not do it anymore, one feature per feature file and everything is fine. Thanks, best regards
